public/js/app.js

```diff
@@ -142,13 +142,19 @@ var makeComment = function(evt){
 
           if(lastComment.length < 1){
             newComment =
-              '<div><h4>1 Responses</h4>'
+              '<div><h4>1 Response</h4>'
             + newComment
             + '</div>';
             form.parent().prev().after(newComment);
+            document.getElementById('comments-project-' + respData.project_id)
+              .innerHTML = 1;
           }
           else {
             lastComment.after(newComment);
+            form.parent().parent().find('h4')
+              .text(respData.commentCount + ' Responses');
+            document.getElementById('comments-project-' + respData.project_id)
```

Comment:
There are a bunch of variables or return values in the existing codebase which use underscore instead of camel case, and in general, there is a mixture of both in some files e.g. PostComment.php

Refactoring such identifiers should be done in its own issue. I just used an existing one in this case so as not to accidentally break something else that might be using that return value.